### PR TITLE
Create database seeds

### PR DESCRIPTION
**Pull request by cballrun**
Adds Events, Artists, Sponsors, Vendors, and join tables for their relationships.

Should be getting sponsor logos from Mike tomorrow. I'll set tasks on the sponsor board for things that need to be updated in the seed file before we put this into production.

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "http://192.168.1.68/graphql"
```
Comment (cgaddis36) [inline config/initializers/cors.rb:10]: Now I know your IP Address!!!! =)

Comment (cgaddis36) [inline config/initializers/cors.rb]: I might modify this to localhost for your commit and modify the one your end as necessary for testing

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,107 @@
...
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.fi
+cfac_2021 = Event.create!(name: "Crawfish for a Cause 2021", city: "Atlanta", state: "Georgia", location: "175 Le Brun Road", description: "The one that started it all at LeBrun")
+cfac_2022 = Event.create!(name: "Crawfish for a Cause 2022", city: "Atlanta", state: "Georgia", location: "Steady Hand Brewery", description: "The first one at Steady Hand.")
+mental_awakening = Event.create!(name: "Mental Awakening", city: "Atlanta", state: "Georgia", location: "Steady Hand Brewery", description: "Mental Awakening 2023")
+cfac_clt = Event.create!(name: "Crawfish for a Cause Charlotte", city: "Charlotte", state: "North Carolina", location: "Suffolk Punch Brewery", description: "The one in Charlotte")
```
Comment (cgaddis36) [inline db/seeds.rb:4]: are these the seeds descriptions you want to display for the site or will these be modified at a later date?

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,107 @@
...
+pluto_gang = Artist.create!(name: "Pluto Gang", logo: "plutoGangLogo", spotify_url: "https://open.spotify.com/artist/4tkYWPJLnt62UiFgek0LlU?si=3e26108b0f18469a", instagram_url: "https://www.instagram.com/plutogangband/?hl=en")
+orange_constant = Artist.create!(name: "Orange Constant", logo: "orangeConstantLogo", spotify_url: "https://open.spotify.com/artist/2ZJzTxzesjXqxREe1J7YYM?si=c40c0054a4b240bd", instagram_url: "https://www.instagram.com/orangeconstant/?hl=en")
+mountain_grass_unit = Artist.create!(name: "Mountain Grass Unit", logo: "mountainGrassLogo", spotify_url: "https://open.spotify.com/artist/0FWlJ725NEZpxqxjZG3yGl?si=c04e3257838b4905", instagram_url: "https://www.instagram.com/the_mountain_grass_unit/?hl=en")
+airshow = Artist.create!(name: "Airshow", logo: "airshowLogo", spotify_url: "https://open.spotify.com/artist/2USLUiPGlv1amJy3gUszGw?si=s63E-FG5QmShLmsnVqCD5A", instagram_url: "https://www.instagram.com/airshowband/?hl=en")
+fixins = Artist.create!(name: "The Fixin's", logo: "fixinsLogo", spotify_url: "https://open.spotify.com/artist/1vJ6XtIbCrUJmoqk2a6kV9", instagram_url: "https://www.instagram.com/thefixinsmusic/?hl=en")
```
Comment (cgaddis36) [inline db/seeds.rb:12]: nice work on Artists

File: db/seeds.rb [hunk 4]
```diff
@@ -1,7 +1,107 @@
...
+reliable = Sponsor.create!(name: "Reliable Restoration", description: "", logo: "random logo") 
+steady_hand = Sponsor.create!(name: "Steady Hand Brewery", description: "", logo: "random logo") 
+eargasm = Sponsor.create!(name: "Eargasm Earplugs", description: "", logo: "random logo")  
+barrys = Sponsor.create!(name: "Barry's", description: "", logo: "random logo") 
+supreme_lending = Sponsor.create!(name: "Supreme Lending", description: "", logo: "random logo") 
+suffolk_punch = Sponsor.create!(name: "Suffolk Punch Brewing", description: "", logo: "random logo")
+
+blake_house = Sponsor.create!(name: "The Blake House", description: "", logo: "random logo")
+will_to_live = Sponsor.create!(name: "Will to Live", description: "", logo: "random logo")
+youth_villages = Sponsor.create!(name: "Youth Villages", description: "", logo: "random logo")
```
Comment (cgaddis36) [inline db/seeds.rb:52]: add description later

**Review by cgaddis36 — COMMENTED** [review]
Nice work on the seeds. Cors file should be modified to remove your ip

File: db/seeds.rb (hunk 2 above)
Comment (cballrun) [reply db/seeds.rb:4]: These descriptions will be modified at a later date. I am collaborating with the co-founder and one of our directors on what we want to include here.

File: db/seeds.rb (hunk 4 above)
Comment (cballrun) [reply db/seeds.rb:52]: I'm working with the directors on this as well. I need to follow up with my co-founder for the logos, too.

File: config/initializers/cors.rb [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins ""
+    origins "localhost:3000/graphql"
```
Comment (cgaddis36) [inline config/initializers/cors.rb:10]: great job fixing this

**Review by cgaddis36 — APPROVED** [review]

**Pull request merged**